test/it/parser/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -2644,4 +2644,144 @@
             </expression-projection>
         </projections>
     </select>
+
+    <select sql-case-id="select_format_pico_time">
+        <projections start-index="7" stop-index="28">
+            <expression-projection start-index="7" stop-index="28" text="FORMAT_PICO_TIME(3501)">
+                <expr>
+                    <function function-name="FORMAT_PICO_TIME" text="FORMAT_PICO_TIME(3501)" start-index="7" stop-index="28">
+                        <parameter>
+                            <literal-expression value="3501" start-index="24" stop-index="27" />
+                        </parameter>
+                    </function>
+                </expr>
+            </expression-projection>
+        </projections>
+    </select>
+
+    <select sql-case-id="select_found_rows">
+        <projections start-index="7" stop-index="18">
+            <expression-projection start-index="7" stop-index="18" text="FOUND_ROWS()">
+                <expr>
+                    <function function-name="FOUND_ROWS" text="FOUND_ROWS()" start-index="7" stop-index="18">
```

Comment:
Can you use inline `<function />` when there is no parameter tag in function.

Reply:
okey, I'll fix it.